search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -666,3 +666,33 @@
     (when (seq errors)
       (errors/throw-service-errors :bad-request errors)))
   params)
+
+(def ^:private valid-deleted-collections-search-params
+  "Valid parameters for deleted collections search"
+  #{:revision-date
+    :revision_date
+    :result-format})
+
+(defn- unrecognized-deleted-colls-params-validation
+  "Validates that no invalid parameters were supplied to deleted collections search"
+  [params]
+  (map #(format "Parameter [%s] was not recognized." (csk/->snake_case_string %))
+       (set/difference (set (keys params)) valid-deleted-collections-search-params)))
+
+(defn- deleted-colls-result-format-validation
```

Comment:
Was it in the requirement to return XML? Just curious because it seems like our json response is so much easier to work with.